### PR TITLE
[stable/chaoskube] Support K8S older then 1.9

### PR DESCRIPTION
**Pull request by shaikatz**
Add support for API versions for older k8s in the deployment file.

### CODE REVIEW

**Comment by shaikatz** [comment]
Verify CLA

**Comment by shaikatz** [comment]
Verify CLA

**Comment by linki** [comment]
Could you sign the CLA @shaikatz?

**Comment by shaikatz** [comment]
Verify CLA

**Comment by shaikatz** [comment]
Verify CLA

**Comment by shaikatz** [comment]
Verify CLA

**Comment by shaikatz** [comment]
Verify CLA

**Comment by shaikatz** [comment]
@linki I had some issues signing the CLA, it's ready now :)

File: stable/chaoskube/templates/deployment.yaml [hunk 1]
```diff
@@ -1,4 +1,8 @@
+{{- if semverCompare "^1.9-0" .Capabilities.KubeVersion.GitVersion }}
```
Comment (davidkarlsen) [inline stable/chaoskube/templates/deployment.yaml]: Is it better to use capabilities in https://github.com/helm/helm/blob/master/docs/chart_template_guide/builtin_objects.md: "Capabilities.APIVersions.Has $version indicates whether a version (batch/v1) is enabled on the cluster."

**Comment by shaikatz** [comment]
@davidkarlsen Cool method, wasn't familiar with that.

**Pull request merged**